backend/hostgw/hostgw.go

```diff
@@ -133,7 +137,7 @@ func (rb *HostgwBackend) handleSubnetEvents(batch []subnet.Event) {
 	for _, evt := range batch {
 		switch evt.Type {
 		case subnet.SubnetAdded:
-			log.Infof("Subnet added: %v via %v", evt.Lease.Subnet, evt.Lease.Attrs.PublicIP)
+			log.Infof("Subnet added: %v via %v", evt.Lease.Subnet, evt.Lease.Attrs.InterfaceIP)
```

Comment:
Done, once again, warn me to squash in case you think everything is as it should be. :)